frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-capacity.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import './breakdown-card.scss';
+
+export const TotalCapacityBody: React.FC<TotalCapacityBodyProps> = ({ value, classname }) => {
```

Comment:
Why are we adding prop for class.
I think we will reuse the same one in noobaa with no changes.